### PR TITLE
Commerce 3.3.0

### PR DESCRIPTION
**Pull request by mattstein**
### Description

This branch contains documentation updates for release with [Commerce 3.3.0](https://github.com/craftcms/commerce/pull/2050). ~~Once everything’s finished, I’ll take it out of draft mode and request review here.~~

- [x] controller actions reference
    - [x] add `commerce/cart/complete`
    - [x] update `commerce/payments/pay`
- [x] payment gateway `supportsPartialPayments()`
- [x] partial payments
- [x] order notices
- [x] example template generator
- [x] shipping rules dynamic condition formula
- [x] “Copy cart load URL” order index actions
- [x] new `EVENT_BEFORE_PURGE_ADDRESSES` event
- [x] changed config settings
    - [x] new `allowCheckoutWithoutPayment` 
    - [x] new `allowPartialPaymentOnCheckout`
- [x] console commands
- [x] example template path updates

### CODE REVIEW

**Comment by mattstein** [comment]
Assuming the `buy/` templates are gone now and that `commerce/example-templates` is the only new console command.

**Comment by mattstein** [comment]
Finally ready for review!

The checklist above is a granular list of stuff I updated if my commits weren’t tedious enough. Things you’ll want to scrutinize most: 

- the rejuvenated [Making Payments page](https://github.com/craftcms/docs/blob/release/commerce/3.3.0/docs/commerce/3.x/making-payments.md) that now has a place in the sidebar under Front End Development  
- the [Controller Actions page](https://github.com/craftcms/docs/blob/release/commerce/3.3.0/docs/commerce/3.x/dev/controller-actions.md), which @nfourtythree sharply checked (and corrected) prior to my [updates for partial payments](https://github.com/craftcms/docs/pull/189/commits/5d68bbfcc5d127da0d19c3b610dc651932821809#diff-2c9a7d26dbac40aa7313f1d86a034c4db61da4d92dca1f376a62f57c12b3b54a)

Please comment or approve and I’ll merge when it’s time!

File: docs/commerce/3.x/making-payments.md [hunk 1]
```diff
@@ -141,5 +235,24 @@ If you would like to pay with a different gateway than the one that is already s
...
+
+1. Navigate to **Commerce** → **Orders**.
+2. Choose a cart or order to view its edit page
+3. Select the **Transactions** tab and choose **Make Payment**.
+4. Select a gateway that supports partial payments to enter the payment details and amount:\
+    ![Store manager’s partial payment modal](./assets/partial-payment-modal.png)
+5. Choose **Pay** to process the payment.
+
+::: tip
+To more quickly identify partially-paid orders, a store manager may want to add the **Paid Status** column in the Orders table.
```
Comment (lukeholder) [inline docs/commerce/3.x/making-payments.md]: @mattstein I think Brandon calls it the "Order index page" or "Order index table" and not "Order table"

**Review by lukeholder — APPROVED** [review]
I have read it all and did not see anything incorrect. You've done it again @mattstein

**Comment by mattstein** [comment]
Thanks @lukeholder!

**Comment by jerome2710** [comment]
Could the [gateway configuration](https://craftcms.com/docs/commerce/3.x/payment-gateways.html#storing-config-outside-of-the-database) also be updated? Since the "old" configuration is deprecated, it is not really clear how one should refactor the config. Especially since according to https://github.com/craftcms/commerce/issues/2101 not all gateways were/are ready for this change yet.

**Comment by nfourtythree** [comment]
Hi @jerome2710 

The best thing to do is follow along with the issue you mentioned as this is where we will update on the further developments regarding gateway configuration (docs will be updated as required).

For the moment using the existing gateway configuration method, as outlined in the docs, is still applicable.

Thanks.

**Pull request merged**